app/controllers/admin/stylesheets_controller.rb

```diff
@@ -6,7 +6,7 @@ def show
 
   def create
     Stylesheet.temporary(current_account, params[:url]) do
-      redirect_to "/" + Stylesheet.capture("http://#{MY_DOMAIN}/", cookies)
```

Comment:
To be checked, I'm not sure how the redirect url was formed before

Reply:
If I understand correctly, the goal is to create a screenshot of the homepage, so we can replace "http://#{MY_DOMAIN}/" by "#{MY_PUBLIC_URL}/".

The result of the `Stylesheet.capture` is the name of a temporary file (like `tmp/snapshot_123.png`) without the leading slash character (`/`). So I had to keep the `"/" +` on the line.